lib/ts/recipe/thirdparty/providers/apple.ts

```diff
@@ -132,11 +132,14 @@ export default function Apple(config: TypeThirdPartyProviderAppleConfig): TypePr
             accessTokenAPIParams.redirect_uri = redirectURI;
         }
         let authorisationRedirectURL = "https://appleid.apple.com/auth/authorize";
-        let scopes = ["name", "email"];
+        let scopes = ["email"];
         if (config.scope !== undefined) {
-            scopes.push(...config.scope);
+            scopes = config.scope;
             scopes = Array.from(new Set(scopes));
         }
+        if (!scopes.includes("email")) {
+            throw Error("Scope email is required. Please add it to the scope array");
+        }
```

Comment:
Why are you adding this check if you are allowing the user to pass their own `scope`? If you are doing this, you might as well add this to their scope too.. I think you should remove this `if` statement completely (applies for all providers).